Makefile

```diff
@@ -42,6 +42,27 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+PLATFORMS := darwin linux windows
+
+WIN_FLAGS = -ldflags "-X main.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
```

Comment:
Why both internal and external linkflags?

Reply:
my best guess would be staying close to the regular linkflags (aka copy & paste remnant) while being able to compile a windows binary